kt/godot-library/src/main/kotlin/godot/tests.kt

```diff
@@ -337,20 +335,30 @@ open class Node : Object() {
     open fun getParent(): Node? {
         TransferContext.writeArguments()
         TransferContext.callMethod(rawPtr, NODE_GET_PARENT, VariantType.OBJECT)
-        val readReturnValue = TransferContext.readReturnValue(VariantType.OBJECT)
+        val readReturnValue = TransferContext.readReturnValue(VariantType.OBJECT, true)
         return readReturnValue as Node?
     }
 
     open fun addChild(node: Node) {
         TransferContext.writeArguments(VariantType.OBJECT to node)
         TransferContext.callMethod(rawPtr, NODE_ADD_CHILD, VariantType.NIL)
-        TransferContext.readReturnValue(VariantType.NIL)
     }
 
     open fun removeChild(node: Node) {
         TransferContext.writeArguments(VariantType.OBJECT to node)
         TransferContext.callMethod(rawPtr, NODE_REMOVE_CHILD, VariantType.NIL)
-        TransferContext.readReturnValue(VariantType.NIL)
+    }
+
+    open fun getNode(nodePath: NodePath): Node? {
```

Comment:
I think we should also provide the same method with a string as a param and internally wrap that string into a nodePath for convenience

Reply:
This will be generated with json api, maybe we should do extention function after api generation.

Reply:
Ok for me if we do it there